tests/foreman/api/test_ansible.py

```diff
@@ -59,3 +63,65 @@ def test_fetch_and_sync_ansible_playbooks(target_sat):
     )
     assert task_details[0].result == 'success'
     assert len(task_details[0].output['result']['created']) == playbooks_count
+
+
+@pytest.mark.e2e
+@pytest.mark.no_containers
+@pytest.mark.rhel_ver_list([7, 8, 9])
+def test_positive_ansible_job_on_host(target_sat, module_org, rhel_contenthost):
+    """
+    Test successful execution of Ansible Job on host.
+
+    :id: c8dcdc54-cb98-4b24-bff9-049a6cc36acb
+
+    :Steps:
+
+        1. Register a content host with satellite
+        2. Import a role into satellite
+        3. Assign that role to a host
+        4. Assert that the role was assigned to the host successfully
+        5. Run the Ansible playbook associated with that role
+        6. Check if the job is executed.
+
+    :expectedresults:
+        1. Host should be assigned the proper role.
+        2. Job execution must be successful.
+
+    :CaseAutomation: Automated
```

Comment:
CaseAutomation already defined at module level, this can be removed

Reply:
I want to keep this one in the future if I have nonautomated scenarios It's beneficial to have a count for coverage.